### DISCUSSION TITLE
some jxr files problem on ubuntu

### DISCUSSION THREAD

**Discussion by Zimovets** [ImageMagick/ImageMagick#7286] [Help]
### ImageMagick version

7.1.1

### Operating system, version and so on

Ubuntu 22.04.4 LTS

### Description

I worked with imagemagick in my nodejs project, i try to convert .jxr into jpeg, and strange thing i had file to test and it worked fine but when i try another file i got error
[jxr_files.zip](https://github.com/ImageMagick/ImageMagick/files/15198174/jxr_files.zip)
:

Error: Command failed: Unrecognized WMPTag: 11(0xb), 2, 20, 0xf2
Unrecognized WMPTag: 42032(0xa430), 2, 27, 0x1f0
Unrecognized WMPTag: 50708(0xc614), 2, 30, 0x21c
Unrecognized WMPTag: 50934(0xc6f6), 2, 27, 0x23a
convert-im6.q16: unable to open image `/tmp/magick-dCij_pHyI6Cc6x9QTlRHSDg0d3FJvuQb': No such file or directory @ error/blob.c/OpenBlob/2924.
convert-im6.q16: unable to open file `/tmp/magick-dCij_pHyI6Cc6x9QTlRHSDg0d3FJvuQb': No such file or directory @ error/constitute.c/ReadImage/614.
convert-im6.q16: no images defined `/home/pc/projects/hillel/hillel-api/uploads/temp/temp_1714727358346.jpg' @ error/convert.c/ConvertImageCommand/3229.

    at ChildProcess.<anonymous> (/home/pc/projects/hillel/hillel-api/node_modules/imagemagick/imagemagick.js:88:15)
    at ChildProcess.emit (node:events:514:28)
    at ChildProcess.emit (node:domain:489:12)
    at maybeClose (node:internal/child_process:1091:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
    at Process.callbackTrampoline (node:internal/async_hooks:130:17) {
  timedOut: false,
  killed: false,
  code: 1,
  signal: null
}

i attached zip with both files, for_test.jxr work another throw the error, i noticed there is a big diff in file size

**Comment by dlemstra** [comment]
Looking at the error message it seems that one of your tests images is corrupt and cannot be read and that is why it doesn't work?

**Reply by Zimovets** [reply]
the same image works perfact on the macos, have problem only on my ubuntu os

**Reply by dlemstra** [reply]
I suspect that you have two different versions of that application. And there is not much we can do about this.
